CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ These are the section headers that we use:
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
Shouldn't this go under the 1.11.1 if we're planning to release it as a bug-fix?

Reply:
This fix will be released in v1.12.0